deployability/modules/testing/tests/test_agent/test_basic_info.py

```diff
@@ -6,8 +6,8 @@
 import re
```

Comment:
Move this import before pytest because it is a standard library import.

Reply:
Done in  https://github.com/wazuh/wazuh-qa/pull/5310/commits/65ca9b4687cd5d98cb49b609aa5d1f9a662d666c